nntrainer/tensor/manager.h

```diff
@@ -334,7 +337,11 @@ class Manager {
   unsigned int max_grad_size; /**< max trainable weight required by a layer */
   unsigned int max_derivative_size; /**< max derivative required by a layer */
   unsigned int max_shared_inout;    /**< max memory for in/outs for inference */
+
   bool weights_initialized; /**< track if weights have been initialized */
+  bool tensors_initialized; /**< track if other tensors have been initialized */
```

Comment:
Q. what does mean by others? Could you classify by derivative + in_outs + gradients?

Reply:
Self answered: yes